Makefile

```diff
@@ -0,0 +1,40 @@
+# Runs unit tests for `mediapipe_core`
+test_core:
+	cd packages/mediapipe-core && dart test
+
+# Runs the utility to pull in all header files from `google/mediapipe`
+headers:
```

Comment:
This is not used by any of the other commands, is it a prereq for any of the others?

Reply:
No, it's just a standalone command to document / easily trigger the headers aggregation task.